src/utils/components/AlertsCard/AlertsDrawer.tsx

```diff
@@ -63,7 +67,18 @@ const AlertsDrawer: React.FC<AlertsDrawerProps> = ({ sortedAlerts }) => {
                 }
 
                 return (
-                  <span key={alertType}>
+                  <Button
+                    variant="plain"
+                    className="pf-m-link--align-left"
+                    key={alertType}
+                    onClick={(e) => {
+                      setAlertOpen(
+                        titleOpen && alertOpen === alertType ? null : (alertType as AlertType),
+                      );
+                      setTitleOpen(!titleOpen || alertOpen !== alertType);
```

Comment:
when changing based on previous state, to guarantee getting the last state we should follow this pattern:
`setState((prevState) => { // do something; return newState });`